pymeasure/instruments/etslindgren/emcenter.py

```diff
@@ -0,0 +1,83 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+
+import logging
+import re
+from datetime import datetime, timezone, timedelta
+import numpy as np
+import pandas as pd
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import (
+    strict_discrete_set, strict_discrete_range,
+    truncated_discrete_set, strict_range
+)
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+
+
+
+
+class EMCenter(Instrument):
+    """ Represents ETS-Lindgren EMCenter main frame.
+        used for direct access to the instrument, does not control sub-modules
+
+        
+
+       
+
+    
+    """
+
+    local = Instrument.control(
+            'LOCAL',
+            """ Sets the EMCenter to local mode.""",
+    )
+    reboot = Instrument.control(
+            'REBOOT SYSTEM',
+            """ Reboots the EMCenter.""",
+    )
```

Comment:
I'd keep reboot as a method, as you call it and don't want to set "reboot" to a value, nor want to read the value of "reboot".